client/src/components/FilterSelect.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type FilterSelectProps = {
+  handleChange?: (e: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
It's `handleChange` an optional prop? If it's it doesn't have a default value on the function params destructuring or through `component.defaultProps`

Reply:
it shouldn't have been optional. My mistake